mcp_server.py

```
@@ -0,0 +1,295 @@
+    if target is None or not target.exists():
+        return {
+            "found": False,
+            "name": name,
+            "content": "",
+            "error": f"Document '{name}' not found. Call list_docs() to see available pages.",
+        }
+
+    try:
+        content = target.read_text(encoding="utf-8")
+        return {"found": True, "name": name, "content": content, "error": ""}
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)